cysync/src/store/hooks/flows/useAddCoin.ts

```diff
@@ -123,17 +123,17 @@ export const useAddCoin: UseAddCoin = () => {
   };
 
   // This starts the adding coin task in a queue similar to `syncProvider`.
-  const setUpCoinWallets = async (xpubList: Xpub[]) => {
+  const setUpCoinWallets = async (coinList: Coin[]) => {
     const coinStatus: AddCoinStatus[] = [];
 
     let i = 0;
-    for (const xpub of xpubList) {
-      const coin = COINS[xpub.coin];
-      if (!coin) {
-        throw new Error(`Cannot find coinType: ${xpub.coin}`);
+    for (const coin of coinList) {
```

Comment:
write like this, you'll get index too 
```
for ([index,value] of coinList.entries()) {
    // code
}
```

Reply:
this looks nice.
@irshadCypherock there are other places too that needs replacing the conventional for-loops like for(i =0; i< len; ++i) . Please make a note and update it as you see